src/libraries/System.Private.CoreLib/src/System/Globalization/CultureData.iOS.cs

```diff
@@ -0,0 +1,59 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Text;
+using System.Runtime.InteropServices;
+
+namespace System.Globalization
+{
+    internal sealed partial class CultureData
+    {
+        // Native constants, check if we need this for native
+        private const int Native_ULOC_KEYWORD_AND_VALUES_CAPACITY = 100; // max size of keyword or value
+        private const int Native_ULOC_FULLNAME_CAPACITY = 157;           // max size of locale name
+
+        /// <summary>
+        /// This method uses the sRealName field (which is initialized by the constructor before this is called) to
+        /// initialize the rest of the state of CultureData based on the underlying OS globalization library.
+        /// </summary>
+        private bool InitNativeCultureDataCore()
+        {
```

Comment:
How does this get invoked?

Reply:
This will be invoked in later stages, when more functions will be implemented.
For now I need to load most of the features with icu and only call these 2 functions(NativeGetLocaleName and NativeGetLocaleInfoString) implemented in this PR.